integration/spark/build.gradle

```diff
@@ -60,7 +60,10 @@ ext {
     junit5Version = '5.7.2'
     bytebuddyVersion = '1.10.21'
     sparkVersion = '2.4.7'
-    jacksonVersion = '2.12.2'
+    jacksonVersion = '2.6.7'
```

Comment:
we can use spark 3 jackson version: 2.12.5 it's one for all dependencies and more recent
https://github.com/apache/spark/blob/6745d778184a7ecdff77458d0bca65d3f637affe/pom.xml#L170
Want to use spark3 version when this PR will go to main https://github.com/OpenLineage/OpenLineage/pull/248

Reply:
> Want to use spark3 version when this PR will go to main #248

Are you suggestion we use ` 2.12.5` for `spark3` once this PR has been merged?

Reply:
these versions for spark 2.4.7 tests, and for spark 3.1 test we will use 2.12.5 version